.travis.yml

```diff
@@ -13,6 +13,10 @@ env:
   global:
     - AWS_KEY=AKIAI6YF75KDZNQJBFXA
     - secure: qOF4BXNkfWCOSqZ1X+HfeToxV2YxsAeFgdJD+aY39UU6LMG800XJIRlIFbDl6MDauFsYo3DMzxIxClqN5VNSV0/5x0Yrx0oMMEMl1WHO1AIJE0u3M18v70wROUW6O7Z4+eSkh9cAVAqpo2saAmaC7Vi2Jz1UlWbhTTd2CIKWSyKZgn6ZlzE8WC55gRtwwH4NlJixDSrwe/uReRdfvoH/WhWjpLxd1WeZGGpn2LuciciKc3Y0cc//vmz2E216OXmNW2veA8AU12BMPHi1PSaA10F8NxYnljmPOU2wAQDb4wHej0Hd9cQjz53365PJ8PpJNbZepYSjufEasdnW3fzO1TRbXi8TafuMcEKbs6k6ocVMyk8XMs0ZbLOavKnh+6Vhf++KAlWKKhXCdl2GQ2f/1B0uRHAH08ph4jREBK3AdBepZK/CeXL7Ti7+u9vFFC++sHtz1qFQpjv+k5Ip15KtbMDqfDASMDJdH2h3o/dLHpz9+tCE3fFMuRet/aRnsjJkULful83ywDlfDfX+79G/JJZum6gA/OCCFm8s8l+SEKz8zqMyYcYgEHyBad062GjYpZ6vWVv4gAgpQKLcICVsyoWt2OWNRv7AUn19bQgI1bxL/lFGjRlWBXrF0zQc4axC0VqoaHaq/Hcy7DnT7Ww9vjeBnLWEyu0DOFftpXNdC2Y=
+
+before_deploy:
+  - if [ "$TRAVIS_BRANCH" == "staging" ]; then mkdir -p dist; cp branded-checkout.html dist/index.html; fi
```

Comment:
`CopyWebpackPlugin ` will also accomplish this https://github.com/CruGlobal/give-web/blob/74d029f127181acfaeaa5212e81f8aea48b81c6e/webpack.config.js#L55. I don't know that having it on prod would hurt but if we only want it on staging this is fine.

That exact config for `unsupportedBrowser.html` looks like a copy paste from the conf-registration-web repo... https://github.com/CruGlobal/give-web/pull/612/files doesn't entirely make sesnse to me anymore. Idk when it would be displayed. Maybe AEM uses it...